hw/top_englishbreakfast/data/top_englishbreakfast_rnd_cnst_pkg.sv.tpl

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+${gencmd}
```

Comment:
is there any reason why this one couldn't be a symlink also...? 
is it just a matter of us changing the original one to not hard code `top_earlgrey` in its package name?

Reply:
You are absolutely right. With little work, also this template could be replaced by a symlink. I will look into it.

Reply:
This has now been replaced by a symlink, too.